connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -117,41 +119,33 @@ def __get_cosign_validated_digests(self, image: str, key: str):
             raise UnexpectedCosignData(message=msg)
         return digests
 
-    def __invoke_cosign(self, image, key):
+    def __cosign_callback(self, image: Image, key_args: list):
         """
-        Invoke the Cosign binary in a subprocess for a specific `image` given a `key` and
-        return the returncode, stdout and stderr. Will raise an exception if Cosign times out.
+        Callback function that is passed on to any `key.verify()` that is using a cosign
```

Comment:
Docstring comments should be commands

Should first say what it does and if necessary, then what the context is